packages/editor/src/editor/browser/cms/AddCmsQuickAction.tsx

```diff
@@ -6,17 +6,18 @@ import { useAppContext } from '../../../context/AppContext';
 import { useFunction } from '../../../context/useFunction';
 import { useQueryClient } from '@tanstack/react-query';
 import { genQueryKey } from '../../../query/query-client';
+import type { InputTextAreaRef, SavedSelection } from './addCmsQuickAction-utils';
 
 export const AddCmsQuickActionPopover = ({
   value,
   onChange,
   savedSelection,
-  inputRef
+  inputTextAreaRef
```

Comment:
why this rename? IMO the text area is an input to but the input is no textare. 
Can't you just use the parent interface of both as ref type?